### ISSUE TITLE
Multiple archive targets

### ISSUE THREAD

**Issue by lyncodev** [ericaltendorf/plotman#659]
**Describe the request**
After reading the documentation, it doesn't seem possible to target multiple archiving nodes with rsync.

**Comment by Spaikys** [comment]
I’m setting up second farmer pc and was wondering about same thing, is it possible to use rsynced for plot translate to more then one pc. Without disconnecting first farming pc from plotman.

**Comment by altendky** [comment]
No, you can not have multiple targets such as multiple harvesters at present.  I imagine someday getting to a more flexible archival setup that would allow that, but it's not something I have expected in the near term.

Is it difficult to just connect to one until it is full then connect to the next?

**Comment by Spaikys** [comment]
> No, you can not have multiple targets such as multiple harvesters at present. I imagine someday getting to a more flexible archival setup that would allow that, but it's not something I have expected in the near term.
> 
> Is it difficult to just connect to one until it is full then connect to the next?

It’s not to difficult obviously, that’s what I’m going to do, but I was just wondering is it possible, no I go the answer that it is not for now. Thank you
